awesome-nostr-japan.toml

```
@@ -934,11 +942,20 @@ caption = "Books/Articles"
 
   [BooksArticles.Hello_Nostr]
   name = "Hello Nostr! 先住民が教えるNostrの歩き方"
-  address = "https://nip-book.nostr-jp.org/book/1/"
+  address = "https://techbookfest.org/product/veGeNq1tNkYFfWnZWgEC72?productVariantID=u5RZNM5e77xHH97c65PCx9"
```

```suggestion
  address = "https://nip-book.nostr-jp.org/book/1/"
```
nip-bookのページのほうがいい気がしますが、変更した理由あります?